databases/mongo.md

```diff
@@ -26,17 +26,14 @@ require 'sinatra'
 require 'mongo'
 require 'json/ext' # required for .to_json
 
-include Mongo
-
 configure do
-  conn = MongoClient.new("localhost", 27017)
-  set :mongo_connection, conn
-  set :mongo_db, conn.db('test')
+  db = Mongo::Client.new([ '127.0.0.1:27017' ], :database => 'test')  
+  set :mongo_db, db
 end
 ```
 
-With a connection "in hand" you can connect to any database and collection in
-your Mongo instance.
+After creating a client to a MongoDB server, you can use the driver to
+communicate with the database.
 
 ```ruby
 get '/collections/?' do
```

Comment:
I think this `collections` handler also requires some changes: https://www.dropbox.com/s/5tjdxmkdqrw0k9i/Screenshot%202015-06-29%2020.30.09.png?dl=0